algosdk/v2client/algod.py

```diff
@@ -483,6 +483,47 @@ def get_block_hash(self, round_num, **kwargs):
         req = "/blocks/{}/hash".format(round_num)
         return self.algod_request("GET", req, **kwargs)
 
+    def simulate_transactions(self, txns, **kwargs):
```

Comment:
nit: type-hints would be nice for the `simulate_*` . Even though the rest of the file isn't conforming here, it should be okay to have new methods with annotations and mypy should be able to handle it fine (though you may need to sprinkle some `# type: ignore`'s in the method bodies.

Reply:
Can we make an issue to fully type hint this? feels weird to do just the one method here

Reply:
I support that idea! Shall I create a child PR, or you want to take this?

Reply:
have at it! thank you!

Reply:
Oh well. This dragged in too many un-related changes. So I ended up making a PR into `develop`:

https://github.com/algorand/py-algorand-sdk/pull/442

I can also add in the ATC re-ordering so we don't have to stringify types (see another comment above)